lisp/ess-tracebug.el

```diff
@@ -94,6 +155,44 @@ The postfix keys are defined in `ess-tracebug-map':
 ;; Electric debug keys are now on C-c and ess-dev maps." "ESS 13.05")
 ;; (define-obsolete-variable-alias 'ess-tracebug-command-prefix 'ess-tracebug-prefix)
 
+(defcustom ess-tracebug-search-path nil
+  "List of directories to search for source files.
+Elements should be directory names, not file names of directories.
+"
+  :type '(choice (const :tag "Unset" nil)
+                 (repeat :tag "Directory list" (string :tag "Directory")))
+  :group 'ess-debug)
+
+(defvar ess-watch-buffer "*R watch*"
+  "Name of the watch buffer.")
+
+(defcustom ess-watch-height-threshold nil
```

Comment:
Is there a technical reason to move these defcustoms around? Tracebug file contains a bunch of isolated functionality, each section (like "watch") is self contained. It's more modular in my eyes this way.

Reply:
They were defined after getting referenced so the byte compiler would issue a warning about referencing undefined global variables. My personal preference for elisp files is to have requires, defvars/customs, and defuns in that order, but I can move them further down (but still above their original location) if you want.

Reply:
There are some differences in this regard betwen emacs 27 and others I think. Ok.